Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,423 @@
+<?php
+
+/**
+ * File containing the main context class for PlatformUI
+ * This flie contains the mapping of the BDD sentences to the functions that implement them
+ * Also contains the necessary initializations
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context;
+
+use EzSystems\BehatBundle\Context\Browser\Context;
+use EzSystems\PlatformUIBundle\Features\Helper\JavaScript as JsHelper;
+use Behat\Gherkin\Node\TableNode;
+use PHPUnit_Framework_Assert as Assertion;
+
+class PlatformUI extends Context
+{
+    use SubContext\Authentication;
+    use SubContext\CommonActions;
+
+    /**
+     * PlatformUI relative URL path
+     *
+     * @var string
+     */
+    private $platformUiUri;
+
+    /**
+     * WebDriver in use
+     * Only Sahi or Selenium2 accepted
+     *
+     * @var string
+     */
+    protected $driver;
+
+    /**
+     * Evaluate javascript code and return result.
+     *
+     * @param   string  $jsCode
+     * @return  mixed
+     */
+    protected function evalJavascript( $jsCode, $addJsHelper = true )
```

Comment:
most likely we should be able to get rid of this method and `execJavascript` but one thing though, with `$addJsHelper` set to true (default value), the whole `PlatformUI.js` is read and injected in the page again and again.